packages/native/webpack.browser.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path');
+const { mergeCommon } = require('./webpack.common');
 
-module.exports = {
+module.exports = mergeCommon({
```

Comment:
Apparently Webpack has already thought of this. There's

```javascript
const { merge } = require('webpack-merge');
```

[link](https://webpack.js.org/guides/production/)

Although I like the idea of not putting a new thing to our dependencies and your `mergeCommon` does the exact same thing, I also like the idea of following "official" guidelines.

So I don't know, I'll leave it up to you.

Reply:
Nice, I was not aware of that, I will update it.

Reply:
Or perhaps a simple Object.assign will do?